scripts/benchmark_stats.py

```diff
@@ -48,8 +58,11 @@ def __str__(self):
             f"Percentile Averages:\n"
             f"{formatted_percentiles}\n"
             f"---\n"
-            f"GETs (99th): {self.average_get}ms\n"
-            f"POSTs (99th): {self.average_post}ms\n"
+            f"GETs (99th): {formatted_metrics.get('average_get')}\n"
```

Comment:
Can get rid of the .gets here as well 👍

Reply:
Removed 👍 there are a few others in that file from before, would it be worth removing those? e.g. `row.get("request_count")` or is a chance that it may not exist?

Reply:
Depends if we know that they will always be present